Tasks/S3Upload/TaskOperations.ts

```diff
@@ -67,6 +67,23 @@ export class TaskOperations {
         }
     }
 
+    private getCacheControl(currentFile: string): string {
+        for (const expression of this.taskParameters.cacheControl) {
+            const firstIndex = expression.indexOf('=')
+            if (firstIndex < 1) {
+                continue
```

Comment:
Log the that the cache control entry was in an invalid format. Should we actually fail the task because that means the user set invalid parameters for the task and we would be silently ignoring.